NewForce-Cohort-6/tabloid-cli-coven-of-the-lunar-owls
# Completed "Add Journal Entry #3"---Squashed lots of bugs.

# Description

Fixed a plethora of bugs that caused the window to crash upon trying to run Journal.   
>Fixed Journal Names. 
Added:
    Journal.cs,
    JournalRepository.cs,
    JournalManager.cs,

# Testing Instructions
Testing:
1 - start terminal
2- Under Main menu; select 1) "Journal Management"
3 - Select ##3)Add Journal
4 -  input journal information
-New Journal
-Title:
-Content:

5 - Entry should save and be viewable via a sql query.  (SELECT * FROM Journal)






# Checklist:

- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [x ] My changes generate no new warnings or errors
- [ x] I have added test instructions that prove my fix is effective or that my feature works
